.travis.yml

```diff
@@ -32,18 +33,20 @@ matrix:
 
 before_install:
   - travis_retry composer self-update
-  - if [[ $TRAVIS_PHP_VERSION != "hhvm" && $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini ; fi
+  - phpenv config-rm xdebug.ini || true
 
 install:
   - if [[ $DEPS == 'latest' ]]; then travis_retry composer update --no-interaction ; fi
   - if [[ $DEPS == 'lowest' ]]; then travis_retry composer update --prefer-lowest --prefer-stable --no-interaction ; fi
   - if [[ $DEPS == 'locked' ]]; then travis_retry composer install --no-interaction --ignore-platform-reqs ; fi
-  - composer show
+  - stty cols 120
+  - COLUMNS=120 composer show
```

Comment:
it does, check travis build.
In two words: some travis distros  have term width set to 40 columns, but it should be mostly fixed. 
Check this issue https://github.com/travis-ci/travis-ci/issues/5407